content/release.rst

```diff
@@ -132,180 +132,13 @@ These are the extra steps for checking a release in production:
 Future improvements for the automated release process:
 
 * Create a release notification list and send an email to everyone who cares
-  about new releases. The email should include a changelog for the lastest version.
+  about new releases. The email should include a changelog for the latest version.
   Trac ticket #525.
 * Add a news article to our website
 * Trigger a website crawler to check broken links for download pages and
   documentation.
 
 
-Release Notes
```

Comment:
I think that this should be part of the release process. Yes, the release notes are also used as documentation, but I want this informatio to be part of the release documentation